packages/react/src/Grid/GridCell.tsx

```diff
@@ -7,37 +7,65 @@ import clsx from 'clsx'
 import { ForwardedRef, forwardRef, HTMLAttributes, PropsWithChildren } from 'react'
 import { PageGridColumnNumber } from './PageGrid'
 
-type GridColumns =
+type OneOrThreeGridCellValues =
   | PageGridColumnNumber
-  | {
-      start?: PageGridColumnNumber
-      span: PageGridColumnNumber
-    }
+  | [PageGridColumnNumber, PageGridColumnNumber, PageGridColumnNumber]
 
 export interface GridCellProps extends HTMLAttributes<HTMLDivElement> {
-  gridColumns?: GridColumns
+  fullWidth?: boolean
+  span?: OneOrThreeGridCellValues
+  start?: OneOrThreeGridCellValues
 }
 
-export const gridColumnClassNames = (gridColumns?: GridColumns) => {
-  if (!gridColumns) {
-    return ''
-  }
+export const gridColumnClassNames = (start?: OneOrThreeGridCellValues, span?: OneOrThreeGridCellValues) => {
+  let classes = []
 
-  if (typeof gridColumns === 'number') {
-    return `amsterdam-grid-column-span-${gridColumns}`
+  if (start) {
+    if (typeof start === 'number') {
+      classes.push(`amsterdam-grid-start-${start}`)
+    } else {
+      const [narrow, medium, wide] = start
+
+      classes.push(
+        `amsterdam-narrow-grid-start-${narrow}`,
+        `amsterdam-medium-grid-start-${medium}`,
+        `amsterdam-wide-grid-start-${wide}`,
+      )
+    }
   }
 
-  const { start, span } = gridColumns
+  if (span) {
+    if (typeof span === 'number') {
+      classes.push(`amsterdam-grid-span-${span}`)
+    } else {
+      const [narrow, medium, wide] = span
```

Comment:
So this way you'll never be able to have variations in span across screen sizes?